tweaks/angelsmods/1_update.lua

```diff
@@ -2,5 +2,7 @@ if mods["angelsbioprocessing"] then
 	sctm.tech_dependency_add("bio-temperate-farming", "sct-bio-science-pack")
 	sctm.tech_dependency_add("bio-swamp-farming", "sct-bio-science-pack")
 	sctm.tech_dependency_add("bio-desert-farming", "sct-bio-science-pack")
+	sctm.tech_dependency_add("bio-processing-blue", "bio-processing-green")
```

Comment:
This isn't really needed, blue has red as dependency and red has green.

(but as said, it probably should be fixed in seablock anyway)